projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts

```diff
@@ -28,6 +28,10 @@ export class OnboardingTenantListComponent
   rowData = [];
   tenants: any;
   leads: any;
+  // for Pagination
+  pagination = true;
+  paginationPageSize = 5;
+  paginationPageSizeSelector = [5, 10, 20, 50, 100];
```

Comment:
this is not to be done like this 
we need performance improvement.. based on the pagination value only those many rows should be fetched from backend

Reply:
same everywhere

Reply:
removed